packages/ui/src/components/Chip.svelte

```diff
@@ -33,44 +37,69 @@
   }
 </script>
 
-<div class="flex items-center font-medium-14 max-w-60 p-1 chip">
-  <span class="px-2 overflow-label">{label}</span>
-  <ButtonIcon
-    bind:this={buttonRef}
-    kind="tertiary"
-    size="min"
-    icon={IconClose}
-    inheritColor={true}
-    on:click={() => dispatch('remove')}
-    on:keydown={handleBackspace}
-  />
+<div
+  class="flex items-center font-medium-14 max-w-60 chip {size}"
+  class:removable={isRemovable}
+  style:background-color={backgroundColor}
+  use:tooltip={showTooltip}
+>
+  <span class="px-2 overflow-label chip-label">{label}</span>
+  {#if isRemovable}
+    <ButtonIcon
+      bind:this={buttonRef}
+      kind="tertiary"
+      size="min"
+      icon={IconClose}
+      inheritColor={true}
+      on:click={() => dispatch('remove')}
+      on:keydown={handleBackspace}
+    />
+  {/if}
 </div>
 
 <style lang="scss">
   .chip {
     position: relative;
     height: var(--global-small-Size);
+    padding: var(--spacing-0_5);
     border: none;
     color: var(--tag-on-accent-PorpoiseText);
     background-color: var(--global-accent-BackgroundColor);
     border-radius: var(--small-BorderRadius);
 
-    &:after {
-      content: '';
-      position: absolute;
-      inset: 0;
-      border-radius: var(--small-BorderRadius);
-      overflow: hidden;
-      background-color: transparent;
-      pointer-events: none;
+    &.removable {
+      &:after {
+        content: '';
+        position: absolute;
+        inset: 0;
+        border-radius: var(--small-BorderRadius);
+        overflow: hidden;
+        background-color: transparent;
+        pointer-events: none;
+      }
+
+      &:hover:after {
+        background-color: var(--global-ui-hover-OverlayColor);
+      }
+
+      &:active:after {
+        background-color: var(--global-ui-active-OverlayColor);
+      }
     }
 
-    &:hover:after {
-      background-color: var(--global-ui-hover-OverlayColor);
+    &.min {
+      height: auto;
```

Comment:
Should we set up fixed `min` height?

Reply:
That’s an interesting question. 
We use `--global-min-Size: 20px (1.25rem)` for height in our Figma layouts. 
But in css variables we have `--global-min-Size: 1rem`

I suggest we keep it as it is for now. We will come back to this component and variable soon anyway.